### DISCUSSION TITLE
'특별 관리 정보'에 엔터가 있을 경우 UI에 보여줘야 할까요?

### DISCUSSION THREAD

**Discussion by WaiNaat** [woowacourse-teams/2023-pium#264] [뜨거운 토론]
가끔 공공데이터 '특별 관리 정보'에 줄바꿈이 들어가는 경우가 있어요(예: [관음죽](https://dev.pium.life/dict/6))

지금은 그 부분을 따로 처리하지 않기 때문에 줄바꿈 대신 일반 공백 한 칸으로 나타납니다.
![image](https://github.com/woowacourse-teams/2023-pium/assets/77872742/4e9c4bc6-37b9-41b0-bc11-c41981eb7df7)

아래 사진은 줄바꿈을 강제로 넣은 사진이에요 
![image](https://github.com/woowacourse-teams/2023-pium/assets/77872742/f1df5649-7692-4fa8-826f-e48994b37679)

아래 사진은 강제로 두 줄로 띄운 모습이에요
![image](https://github.com/woowacourse-teams/2023-pium/assets/77872742/b5c9c3f8-154a-4ced-a582-852db110af20)

1. 지금처럼 줄바꿈은 무시한다.
2. 줄바꿈이 있으면 엔터 한 번으로 보여준다.
3. 줄바꿈이 있으면 엔터 두 번으로 보여준다.

저는 처음에는 2였는데 한 줄만 띄운 걸 보니까 좀 안 이쁜 것 같아서 1이나 3이 마음에 들어요

**Comment by hozzijeong** [comment]
저는 3번이 괜찮은것 같아요! 근데, 받은 데이터들로 문단 구분이 가능할까요?

**Reply by WaiNaat** [reply]
문단이 나눠져 있는 경우에는 받아오는 데이터에 `\n`이 들어있어서
정규표현식의 힘으로 `.trim().split(/(?:\r?\n)+/)` 으로 쪼개면 금방 가능해요(아마도)

**Reply by hozzijeong** [reply]
오 구분자가 있다면 가능할것 같긴 하네요

**Comment by Choi-JJunho** [comment]
저는 3번에 한표입니다!

**Comment by WaiNaat** [comment]
우선 3번으로 해보도록 하겠습니다~
